unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/NotificationComponent/ChatNotificationMessageComponentView.cs

```diff
@@ -177,7 +190,7 @@ public void SetNotificationTargetId(string notificationTargetId)
         this.notificationTargetId = notificationTargetId;
     }
 
-    private void ForceUIRefresh()
+    public void ForceUIRefresh()
     {
         Utils.ForceRebuildLayoutImmediate(backgroundTransform);
```

Comment:
I'm seeing you need to use this `ForceRebuildLayoutImmediate` in several places as each time we show the view or each time a new notification enters... this could happen a lot of times. Have you measured, more or less, the performance impact it has? Be careful with that because I remember the `ForceRebuildLayoutImmediate` could be a bit heavy in terms of performance. Please check if you really need to use it in all the cases where you currently have it!

Reply:
I managed to remove an useless one, the others are needed but not used much, only at the creation